.github/scripts/tests.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 
-set -euxo pipefail
-
 unset CARGO_INSTALL_ROOT
 
+
+## Test --disable-strategies
+## Must be run before `set -euxo pipefail` since this command will fail
```

Comment:
hmm I'd rather `set +e` before that test, and `set -e` after, and keep euxo at the top

Reply:
Thanks, I've moved this test to the bottom of the file so `set +e` does not affect any other test.